src/elements/stepper/stepper/stepper.ts

```diff
@@ -37,8 +37,8 @@ class SbbStepperElement extends SbbHydrationMixin(LitElement) {
 
   /** Overrides the behaviour of `orientation` property. */
   @property({ attribute: 'horizontal-from', reflect: true })
-  public set horizontalFrom(value: SbbHorizontalFrom) {
-    this._horizontalFrom = breakpoints.includes(value) ? value : undefined;
+  public set horizontalFrom(value: SbbHorizontalFrom | undefined) {
```

Comment:
Can't we do it with null (for all changes in this file)?

Reply:
change to null would require more changes (eg. default value for horizontal-from, change the SbbStepValidateEventDetails to match the nullable and so on)

Reply:
@kyubisation Should we change the API to null (as its the basic goal) now or should we do it later?

Reply:
Ideally we do it now. @DavideMininni-Fincons How much effort would this be?

Reply:
Minimal, I just have to touch more lines. See commit [cc89dae](https://github.com/sbb-design-systems/lyne-components/pull/3302/commits/cc89daea6f2edc2152c4672688b02c55e9ab4677)